docs/guides/docker.md

```diff
@@ -0,0 +1,261 @@
+---
+label: Docker automation
+order: -2
+description: Learn how to deploy a Docker stack for common media automation software
+image: /static/tohsaka.gif
+---
+
+# Setting up a Docker automation stack
+
+Docker is a piece of software that allows you to virtualize applications into containers, so that you don't have to deal with dependency
+management and can carry around a solution that's easily deployable anywhere you go.
+
+This guide will cover how to setup a Docker stack for common media automation programs, namely: a torrent client, a usenet client, a
+VPN with a killswitch, Sonarr, Radarr, and Jackett. While we've opted for Jackett here, you could also switch it out for Prowlarr if you'd
+prefer something more flexible. It's compose file is very similar to Radarr/Sonarr.
+
+## Installing Docker
+
+First we need to install Docker onto our system. Windows users can do so by downloading
+[Docker Desktop](https://docs.docker.com/desktop/install/windows-install/), while Linux users can follow the instructions outlined
+[here](https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository), after which they can proceed onto the
+[post-installation steps](https://docs.docker.com/engine/install/linux-postinstall).
+
+## Setting up a Docker network
+
+Now we'll be setting up a network so that our containers can communicate with each other. This step's quite simple, it's just `docker network create guide`. A list of all your current networks can be seen with `docker network ls`.
+
+## Generating a Wireguard configuration file
+
+For maximum privacy and security we'll be tunnelling our usenet and torrent clients through a VPN. Radarr, Sonarr and Jackett however won't go
+through that, as some trackers and indexers might not like it if you send requests from a random non-whitelisted IP.
+
+The process for getting a configuration file is different from provider to provider, so we'd recommend looking around your provider's website,
+forum and support articles if you can't find it immediately.
+
+Once you've generated a configuration file we'd also recommend carrying out port forwarding so that you're connectable.
+
+## Deploying everything using `docker compose`
+
+Now that we've gotten everything ready we can finally set up the stack. Paste the following into a file called `docker-compose.yml`. We'd recommend
+this file being a directory that is purely for this Docker stack, such as `/home/user/projects/automation/`.
+
+We would also recommend sorting your media-related folders in the following way for ease of use:
+- `/home/user/data/torrents/` - where you torrent client will save files.
+- `/home/user/data/usenet` - where your usenet client will save files.
+- `/home/user/data/media` - where Sonarr and Radarr will hardlink files so that they can be read by Plex/Jellyfin/etc. later on.
```

Comment:
seperate this into `/home/user/data/media/tv` and `/home/user/data/media/movie` for sonarr and radarr respectively

```
home/
└── user/
    └── data/
        ├── usenet
        ├── torrents
        └── media/
            ├── tv
            └── movies
```